### PR TITLE
Make docs consistent with example; use different limits so it doesn't…

### PR DESCRIPTION
**Pull request by kdorosh**

### CODE REVIEW

File: docs/content/security/rate_limiting/rulepriority/_index.md [hunk 1]
```diff
@@ -99,14 +99,14 @@ curl -H "x-type: Whatsapp" -H "x-number: 311" --head $(glooctl proxy url)
...
-Run the following a couple times:
+Run the following a couple times; you shouldn't get rate-limited:
 ```shell script
 curl -H "x-type: Whatsapp" -H "x-number: 411" --head $(glooctl proxy url)
 ```
 
 Requests to number `411` have a much higher rate limit, and will not return HTTP 429 unless more than 100 requests
 are sent within a minute. Note that this wouldn't work without rule priority (play around with the server settings
-to test this!) because our requests would match the inner Whatsapp rule (Rule 2) that limits all requests to 2/min,
+to test this!) because our requests would match the inner Whatsapp rule (Rule 2) that limits all requests to 1/min,
```
Comment (ashleywang1) [inline docs/content/security/rate_limiting/rulepriority/_index.md]: wait now this should be 2 again XD

Comment (ashleywang1) [reply docs/content/security/rate_limiting/rulepriority/_index.md]: wait no nevermind!

Comment (kdorosh) [reply docs/content/security/rate_limiting/rulepriority/_index.md]: we spoke offline, its right as is

**Pull request merged**